lib/rack/proxy.rb

```diff
@@ -105,6 +106,7 @@ def perform_request(env)
         http.read_timeout = read_timeout
         http.verify_mode = OpenSSL::SSL::VERIFY_NONE if use_ssl && ssl_verify_none
         http.ssl_version = @ssl_version if @ssl_version
+        http.set_debug_output($stdout)
```

Comment:
Fixed . Added that by mistake. I was using it to debug the proxy requests locally